test/logging/logger_writer.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package logging
+
+import (
+	"bytes"
+	"io"
+	"strings"
+	"sync"
+)
+
+type loggerWriter struct {
```

Comment:
Should these be "prefixedWriter" or "prefixWriter"?

Reply:
My idea to name it `loggerWriter` is to clarify that you build this writer from a logger function